step-ca/provisioners.mdx

```diff
@@ -1077,6 +1108,21 @@ To get a client certificate for a hardware-bound private key on your YubiKey:
    }
    ```
 
+##### Device Attestation for TPM 2.0
+
+While `step-ca` supports the `tpm` attestation format of ACME `device-attest-01`,
+an end-to-end TPM-bound client certificate flow is not available at this time.
```

Comment:
In other parts of the docs we've added some small call-outs to our paid offerings. This might be a good spot for that.